bin/apisix

```diff
@@ -346,8 +285,8 @@ end
 
 local checked_admin_key = false
 local function init()
-    if is_root_path then
-        print('Warning! Running apisix under /root is only suitable for development environments'
+    if env.is_root_path then
+        print('Warning! Running apisix under /root is only suitable for development envments'
```

Comment:
Why use `envments`?

Reply:
Oops, it's the result of full replace from "environ" to "env".